index.html

```diff
@@ -350,10 +346,10 @@ <h3 class="h3-about-me-outer">
             </section>
         </section>
         <section class="form-container">
-            <h3 class="form-text">
+            <div class="form-text">
                 I'm always interested in hearing about new projects, 
                 so if you'd like to chat please get in touch.
-            </h3>
+            </div>  
```

Comment:
[ Optional Changes ] 

- [ ] Kindly make this a heading text because this is meant to be the heading for the Contact Me section. For heading hierarchy purposes you can make it h3 as you did with the other headings. 👍